tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/operation/NewPortletClassDataModelProvider.java

```diff
@@ -1024,7 +1024,7 @@ else if( CREATE_RESOURCE_BUNDLE_FILE_PATH.equals( propertyName ) )
             }
             else
             {
-                return PortletCore.createWarningStatus( Msgs.resourceBundleFilePathEndWithProperties );
```

Comment:
For Maven type project, the prefix also has been hardcode with ".properties" and the langFile's default value is "Language"